pkgs/development/python-modules/phonemizer/default.nix

```diff
@@ -31,6 +32,12 @@ buildPythonPackage rec {
       libespeak = "${lib.getLib espeak-ng}/lib/libespeak-ng${stdenv.hostPlatform.extensions.sharedLibrary}";
       # FIXME package festival
     })
+    # This patch is needed for python3Packages.misaki. See https://github.com/thewh1teagle/espeakng-loader?tab=readme-ov-file#usage-with-phonemizer.
+    (fetchpatch2 {
+      name = "pr191-add-option-to-use-custom-espeak-data-path.patch";
+      url = "https://github.com/bootphon/phonemizer/commit/cc1db4bfaf688fdfb8275fd83d218f06411455e6.patch";
```

Comment:
Have you read the docs? Please append ?full_index=1.

https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#fetching-patches

> GitHub supports appending ?full_index=1 to the URL